src/core/include/mp-units/cartesian_vector.h

```diff
@@ -278,8 +154,112 @@ template<typename Arg, typename... Args>
   requires(sizeof...(Args) <= 2) && requires { typename std::common_type_t<Arg, Args...>; }
 cartesian_vector(Arg, Args...) -> cartesian_vector<std::common_type_t<Arg, Args...>>;
 
+MP_UNITS_EXPORT template<typename T, typename U>
+  requires requires(const T& t, const U& u) { t + u; }
+[[nodiscard]] constexpr auto operator+(const cartesian_vector<T>& lhs, const cartesian_vector<U>& rhs)
+{
+  return ::mp_units::cartesian_vector{lhs._coordinates_[0] + rhs._coordinates_[0],
+                                      lhs._coordinates_[1] + rhs._coordinates_[1],
+                                      lhs._coordinates_[2] + rhs._coordinates_[2]};
+}
```

Comment:
All of the operators and other customization points should be hidden friends as they were before.